src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int dimensionLimit = 100;

```

Comment:
It should be a constant, so use static final keywords and name it according to naming convention with upper case